src/rust/engine/graph/src/lib.rs

```diff
@@ -459,7 +459,7 @@ impl<N: Node> InnerGraph<N> {
       .into_iter()
       .filter_map(move |eid| self.entry_for_id(eid))
       .filter_map(move |entry| match entry.peek(&context) {
-        Some(Ok(item)) => N::digest(item),
+        Some(item) => N::digest(item),
```

Comment:
Github won't let me suggest, but: This could now become just a straight `.filter_map(move |entry| entry.peek(&context)).map(N::digest)`